### PR TITLE
Use formatprovider in StonePounds.ToString()

### PR DESCRIPTION
**Pull request by angularsen**
Stumbled upon this when refactoring WRC. Culture param was never used.

### CODE REVIEW

**Comment by angularsen** [comment]
Let's hold off on this one until #607 is merged, I think the build error is more easily fixed then.

**Comment by tmilnthorp** [comment]
Should do the same for FeetInches

File: UnitsNet/CustomCode/Quantities/Length.extra.cs [hunk 1]
```diff
@@ -191,13 +191,14 @@ public override string ToString()
 
         public string ToString([CanBeNull] IFormatProvider cultureInfo)
         {
-            // Note that it isn't customary to use fractions - one wouldn't say "I am 5 feet and 4.5 inches".
-            // So inches are rounded when converting from base units to feet/inches.
+            cultureInfo = cultureInfo ?? GlobalConfiguration.DefaultCulture;
+
             var footUnit = UnitAbbreviationsCache.Default.GetDefaultAbbreviation(LengthUnit.Foot);
```
Comment (tmilnthorp) [inline UnitsNet/CustomCode/Quantities/Length.extra.cs]: Pass cultureInfo to GetDefaultAbbreviation too

**Comment by tmilnthorp** [comment]
🤣

**Comment by angularsen** [comment]
[Release UnitsNet/4.8.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.8.0)

**Pull request merged**